src/model/database.py

```diff
@@ -0,0 +1,35 @@
+import json
+import os
+from model.table import Table
+from commands.keys import Keys
+from output.exceptions import *
+
+
+class Database:
+
+    def __init__(self, schema_name):
+        self.__get_database_schema__(schema_name)
+        self.name = self.database_object[Keys.DATABASE]
+        self.__validate_name__()
+        self.path = os.path.join(Keys.DATABASE_PATH, self.name)
+
+    def serialize(self):
+        self.__create_database__()
+        self.__create_tables__()
+
+    def __validate_name__(self):
+        if self.name is None:
+            raise WrongParameterError("No database detected")
+
+    def __get_database_schema__(self, schema_name):
```

Comment:
get means return something, u should return here and initialize in constructor. 

Rule 
Avoid making `this.x = value` in another places, try to make this only in constructor. which means thos variables are final.